clients/drcachesim/tests/instr_intervals_example.csv

```diff
@@ -0,0 +1,3 @@
+100,10000,
```

Comment:
It's not. I put them to check if they'd cause any issue since simpoints csv files have other fields after the <start,duration> pair (which we ignore).
The test works the same regardless.